src/com/zombiecastlerush/util/Parser.java

```diff
@@ -0,0 +1,23 @@
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Parser {
+    public static void main(String[] args) throws Exception {
+        new Parser().parse("go North");
+    }
+
+    public static ArrayList<String> parse(String input){
```

Comment:
I think you should take all the inputs and make them lower case. `input = input.toLowerCase()`

Reply:
updated line of code: (input.toLowerCase().split(" ")));